python/sglang/srt/constrained/xgrammar_backend.py

```diff
@@ -112,25 +109,16 @@ def __init__(
     ):
         super().__init__()
 
-        if import_error:
-            logger.warning(
-                f"Ignore import error for the grammar backend: {import_error}"
-            )
-            self.grammar_cache = None
-            return
-
         tokenizer_info = TokenizerInfo.from_huggingface(tokenizer)
-        self.grammar_cache = CachedGrammarCompiler(tokenizer_info=tokenizer_info)
+        self.grammar_cache = GrammarCompiler(tokenizer_info=tokenizer_info)
```

Comment:
Would you like to rename grammar_cache to grammar_compiler?

Reply:
Yes, updated in the most recent commit